recipes/blossalg/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "blossalg" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 65c2dd9b084659a20f338552eeea7dc0226a58b6bfc4e5a3e91475d6ad7ec77d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - future
+
+test:
+  imports:
+    - blossom
+  requires:
+    - pytest
+    - future
```

Comment:
`future` should be picked up by the `run` requirements. It'd be good to remove here to make sure that's the case.